sources/api/types.go

```diff
@@ -57,19 +57,27 @@ func (a *AggregateData) Merge(b *AggregateData) {
 	a.Events = append(a.Events, b.Events...)
 }
 
+type ContainerSpec struct {
+	cadvisor.ContainerSpec
+	CpuRequest    int64
```

Comment:
Actually I've found cpu limit inside of cadvisor cpu spec. Adding the new cpu-related fields will create a mess. I would opt for removing cadvisor dependency in sources api (and completing the below TODO).